tests/test_cluster_obj.py

```diff
@@ -163,7 +163,7 @@ def test_blocked_commands(r):
     These commands should be blocked and raise RedisClusterException
     """
     blocked_commands = [
-        "CLIENT SETNAME", "SENTINEL GET-MASTER-ADDR-BY-NAME", 'SENTINEL MASTER', 'SENTINEL MASTERS',
+        'SENTINEL GET-MASTER-ADDR-BY-NAME', 'SENTINEL MASTER', 'SENTINEL MASTERS',
```

Comment:
@Grokzen I have also added this (unrelated) fix for the breaking CI tests since `CLIENT SETNAME` is no longer a blocked command. Please let me know if its better to have this in a separate PR